game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -121,14 +138,31 @@ class Cell(val id: Int, player: Player, var position: Vector2 = new Vector2(0f,
     targetVelocity - velocity
   }
 
+  def movement(deltaSeconds: Float): Vector2 = {
+    steering.truncate(maxSpeed) * deltaSeconds
```

Comment:
`movement` returns a vector that we're supposed to add to the current `velocity`, so we don't want this to be a vector that's too big (or the cell will move *suuuuuper* fast).

`steering` returns a vector from the player's `position` to its `target`, which means that it could have a magnitude of `1` (if they're close) or `10000` (if they're really far).

Thus, we want to ensure that we add a `movement` vector that has a magnitude of *at most* `maxSpeed` (restricting how much we're adding to the `velocity`). `truncate` is a `Vector2` addon (that I wrote earlier) to restrict the magnitude of a vector *if* it's too big.

So, this is just to engage movement (`velocity += ...`) in the direction the player wants to go (`steering`) with a speed cap (`truncate(maxSpeed)`).

The part that allows the velocity to go above the `maxSpeed` was when we removed the `setter` for `velocity` that prevented it from growing too big.